Dockerfile-classic-api

```diff
@@ -0,0 +1,44 @@
+# arxiv/classic-api
+#
+# Defines the runtime for the arXiv classic API, which provides a metadata
+# query API backed by Elasticsearch.
+
+FROM arxiv/base:0.16.6
+
+WORKDIR /opt/arxiv
+
+# Install MySQL.
+# RUN yum install -y which mysql mysql-devel
+
+# Add Python application and configuration.
+ADD classic-api.py /opt/arxiv/
```

Comment:
Move this to after the pipenv install so the python installed packages can be reused as a layer. 

If the code changes, it invalidates the cache for all following layers.

Reply:
please see updated version